client/state/index.js

```diff
@@ -48,16 +50,9 @@ if ( typeof window === 'object' ) {
 	];
 }
 
-let createStoreWithMiddleware = applyMiddleware.apply( null, middleware );
-
-export function createReduxStore( initialState = {} ) {
-	if (
-		typeof window === 'object' &&
-		window.app &&
-		window.app.isDebug &&
-		window.devToolsExtension
-	) {
-		createStoreWithMiddleware = compose( createStoreWithMiddleware, window.devToolsExtension() );
-	}
+export function createReduxStore( initialState = {}, router ) {
+	console.log( router );
+	middleware = [ router5Middleware( router ), ...middleware ];
```

Comment:
If this function somehow gets called more than once, `middleware` will keep growing incorrectly.